.eslintrc.json

```diff
@@ -7,8 +7,11 @@
     "plugin:@cypress/dev/tests"
   ],
   "rules": {
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/no-useless-constructor": "off",
     "prefer-spread": "off",
-    "prefer-rest-params": "off"
+    "prefer-rest-params": "off",
+    "no-useless-constructor": "off"
```

Comment:
- I removed `no-var-requires`.
- Removing `prefer-rest-params` causes probems due to [this line](https://github.com/cypress-io/cypress/blob/ab570bb78de706074e66ee33517b3137b7c16081/packages/driver/src/cypress/utils.js#L61) among a few more others.
- `no-useless-constructor` is requires since eslint doesn't know about the TS constructor shorthand:

```ts
class User {
  private name: string

  constructor(name: string) {
    this.name = name
  }
}
```

Can be written as:

```ts
class User {
  private name: string

  constructor(private name: string) {
  }
}
```